static/src/js/components/GranuleResults/GranuleResultsTable.scss

```diff
@@ -230,18 +230,18 @@
     }
 
     &--add {
-      color: $color__green;
+      color: $color__active-green;
 
       &:hover {
-        color: $color__green--dark;
+        color: $color__active-green;
```

Comment:
should the hover be the same color as the add?

Reply:
I think this is the green glow border hover and is fine.

Reply:
Ok so I double checked, this is the + icon's color on the table view for granules when hovered, seems consistent with what is on prod.  I don't see why it would need to be a different color.